src/linker/Linker.Steps/SweepStep.cs

```diff
@@ -177,18 +172,9 @@ protected void ProcessAssemblyAction (AssemblyDefinition assembly)
 				Annotations.SetAction (assembly, AssemblyAction.AddBypassNGen);
 				goto case AssemblyAction.AddBypassNGen;
 
-			case AssemblyAction.AddBypassNGen:
-				// FIXME: AddBypassNGen is just wrong, it should not be action as we need to
-				// turn it to Action.Save here to e.g. correctly update debug symbols
-				if (!Context.KeepTypeForwarderOnlyAssemblies || BypassNGenToSave.Contains (assembly)) {
```

Comment:
Can we completely remove the `BypassNGenToSave` hashset? I don't think it's used for anything now.